paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -399,3 +417,159 @@ def paasta_stop(args):
         print(PAASTA_STOP_UNDERSPECIFIED_ARGS_MESSAGE)
         return 1
     return paasta_start_or_stop(args, "stop")
+
+
+def paasta_restart_replica(args):
+    """Restart a specific replica (pod) by deleting it and letting Kubernetes create a replacement."""
+    # Validate required arguments
+    pargs = apply_args_filters(args)
+    if len(pargs) == 0:
+        return 1
+
+    # For replica restart, we need exactly one service/instance/cluster combination
+    if len(pargs) > 1:
+        print(
+            PaastaColors.red("Error: --replica can only be used with a single cluster.")
+        )
+        return 1
+
+    cluster = list(pargs.keys())[0]
```

Comment:
just code-golfing: i'm torn between this and being too clever with `cluster, = pargs.keys()` :p

Reply:
Whoops forgot to commit this one too, now fixed for real!